apps/challenges/views.py

```diff
@@ -237,6 +237,9 @@ def get_all_challenges(request, challenge_time):
         q_params['start_date__gt'] = timezone.now()
     # for `all` we dont need any condition in `q_params`
 
+    # don't return disabled challenges
+    q_params['is_disabled'] = False
```

Comment:
@RishabhJain2018 I have not included it in 'query params' since users should not have the choice to view deleted challenges. Once a challenge is removed by the host, the challenge is hidden from the list of all challenges as you have mentioned in a comment in #1080 . Should the host or participants have the choice to view deleted challenges?

Reply:
I think Rishabh might have meant something else which is not applicable here. Thanks for the clarification @codervivek. 

> Should the host or participants have the choice to view deleted challenges?

No one (else than admin who is montoring the django admin dashboard) should see the disabled challenges.

Reply:
Also, I would recommend you to add tests for every small change that you are doing in the project. It helps you to become a good developer and makes the project more robust to errors.